apis/management.cattle.io/v3/pipeline_types.go

```diff
@@ -116,7 +118,7 @@ type SourceCodeConfig struct {
 	URL                      string `json:"url,omitempty" yaml:"url,omitempty" norman:"required"`
 	Branch                   string `json:"branch,omitempty" yaml:"branch,omitempty" `
 	BranchCondition          string `json:"branchCondition,omitempty" yaml:"branchCondition,omitempty" norman:"options=only|except|all"`
-	SourceCodeCredentialName string `json:"sourceCodeCredentialName,omitempty" yaml:"sourceCodeCredentialName,omitempty" norman:"type=reference[sourceCodeCredential]"`
+	SourceCodeCredentialName string `json:"sourceCodeCredentialName,omitempty" yaml:"sourceCodeCredentialId,omitempty" norman:"type=reference[sourceCodeCredential]"`
```

Comment:
is there a reason yml is represented differently from json? Usually both are consistent.